softlayer/vm/softlayer_vm.go

```diff
@@ -24,7 +24,7 @@ const (
 	softLayerVMtag                 = "SoftLayerVM"
 	ROOT_USER_NAME                 = "root"
 	deleteVMLogTag                 = "DeleteVM"
-	TIMEOUT_TRANSACTIONS_DELETE_VM = 10 * time.Minute
+	TIMEOUT_TRANSACTIONS_DELETE_VM = 60 * time.Minute
```

Comment:
The old ruby cpi code doesn't have a timeout value for checking if the delete transaction has started. It just polls to check until the delete transaction starts. So I think it's better to set a longer time for this time out. What's your comments?

For the problem of delete transaction can't start in a short time, we are still communicating with SL.